packages/mds-audit/service.ts

```diff
@@ -105,6 +105,11 @@ export async function readDeviceByVehicleId(provider_id: UUID, vehicle_id: strin
   }
 }
 
+export async function readEvent(device_id: UUID): Promise<Recorded<VehicleEvent>> {
+  const result: Recorded<VehicleEvent> = await db.readEvent(device_id)
```

Comment:
This seems unnecessary, why not just use `db.readEvent` inline instead of wrapping it in another function call?

Reply:
This was me originally. Trying to implement a thin service layer between the API and the DB. Mostly, this was because many DB methods throw when there are no records found and that was not the behavior I wanted. See `getVehicle` below for example.